priv/templates/service.ex.eex

```diff
@@ -4,6 +4,8 @@ defmodule <%= @module %>.Service do
   <% end %>
   use GRPC.Service, name: <%= inspect(@service_name) %>, protoc_gen_elixir_version: "<%= @version %>"
 
+  def full_name, do: "<%= @service_name %>"
```

Comment:
Takes care of the quotes and avoids surprises by always returning a valid string.

```suggestion
  def full_name, do: <%= inspect(@service_name) %>
```